obs-packages/obs-vaapi/PKGBUILD

```diff
@@ -1,19 +1,19 @@
 # Maintainer: Andrius Lukosevicius <niobium93@gmail.com>
 
 pkgname=obs-vaapi
-pkgver=0.1.0
-pkgrel=1
+pkgver=0.2.0
+pkgrel=2
```

Comment:
along with the increase in pkgver, pkgrel is reset

Reply:
Oh, I missed that, thanks, dude!